js/piwik.js

```diff
@@ -7277,10 +7285,31 @@ if (typeof window.Piwik !== 'object') {
              * want to re-ask for consent after a specific time period.
              */
             this.forgetConsentGiven = function () {
-                deleteCookie('consent', configCookiePath, configCookieDomain);
+                deleteCookie(CONSENT_COOKIE_NAME, configCookiePath, configCookieDomain);
```

Comment:
In `rememberConsentGiven` we log if cookies are disabled. I wonder if we should do the same here or ignore disabled cookies and set it anyway?  I think though that we cannot really ignore `configCookiesDisabled` because `setCookie` checks it.

```
    if (configCookiesDisabled) {
                    logConsoleError('rememberConsentGiven is called but cookies are disabled, consent will not be remembered');
                    return;
                }
```

Reply:
Think checking in `setCookie` is better than everywhere it's used, otherwise we'll just have to keep adding the same check.

Reply:
👍 Only meant to add a log message. There will be users who implement the opt out themselves but have cookies disabled. Then it will be useful to log that it won't work.